doc/sphinx/conf.py.in

```diff
@@ -242,8 +242,9 @@ latex_elements = {
 # (source start file, target name, title,
 #  author, documentclass [howto, manual, or own class]).
 latex_documents = [
-  (master_doc, 'ExtensibleSimulationPackageforSoftMatterResearch.tex', 'Extensible Simulation Package for Soft Matter Research Documentation',
-   'Florian Weik', 'manual'),
+    (master_doc, 'ExtensibleSimulationPackageforSoftMatterResearch.tex',
```

Comment:
I don't know who introduced this but this is the wrong name for the package

Reply:
Thanks! Fixed it there and in a few other places.